_posts/2021-09-26-where-the-store-is-stored.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Where is my store stored?
+
+draft: false
+
+page : false
+
+image: /assets/img/where-is-store-stored-illustration.png 
+
+excerpt: >
+  Not all stores location are equal in our our Javascript codebases. Let's look
+  at them to understand the constraints that come with them and how they impact
+  how our stores can be used.
```

Comment:
I propose this that is hopefully a bit more catchy @Benrajalu @asauray, wdyt?

Reply:
`how a store's location impacts how it can be used.`